netket/graph/lattice.py

```diff
@@ -246,6 +315,114 @@ def atoms_coord(self):
         """
         return self._atoms_coord
 
+    @property
+    def coords(self):
+        """
+        Returns list of coordinates of lattice points
+        """
+
+        return self._coords
+
+    def translation_perm(self):
+        perms = []
+        for vec in self.basis_vectors:
+            perm = []
+            for coord in self._coords:
+                hash_coord = coord.copy() + vec
+                hash_coord = _np.matmul(hash_coord, self._inv_dims) % 1
+                # make sure 1 and 0 are treated the same
+                hash_coord = _np.around(hash_coord * 10 ** tol_digits).astype(int) % (
+                    10 ** tol_digits
+                )
+                hash_coord = HashableArray(hash_coord)
+                perm.append(self._hash_positions[hash_coord])
+
+            perms.append(tuple(perm))
+        return tuple(perms)
+
+    def rotation_perm(self, period, axes=[0, 1]):
```

Comment:
Like above.
Type annotations for inputs/output. Docsrting. and if internal and not meant for public API prepend with an underscore.